openapi3.yaml

```diff
@@ -207,8 +208,6 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/tasksResponse'
-        204:
```

Comment:
if that was actually implemented removing it is breaking change

Reply:
I didnt say it wrong.
I said its breaking and you should create tasks for fixing the broken client